src/accordion/accordion.ts

```diff
@@ -88,7 +95,8 @@ export interface NgbPanelChangeEvent {
   </div>
 `
 })
-export class NgbAccordion implements AfterContentChecked {
+export class NgbAccordion implements AfterContentChecked,
+    OnInit {
```

Comment:
Why do we have `OnInit` imported and used in `implements` here? I don't see it being used (I would expect TS to complain in this case...)